src/data/chainDetails.json

```diff
@@ -799,5 +835,158 @@
 			"url": "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/eth.svg",
 			"format": "svg"
 		}
+	},
+	"97": {
```

Comment:
Is this a newly deployed chain?

Reply:
bnb testnet. we're deployed but don't bother adding to ui. it just got populated when i ran fillChainDetails